internal/config-reloader/main.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 var (
+	mode                   = flag.String("mode", "", "default is normal. If using one-shot mode, will read config and write to config-envsubst-file once")
```

Comment:
Not sure about the mode name too :)

Reply:
I think, boolean flag should be enough  `only-init-config`.